core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -25,18 +25,27 @@ import javax.xml.transform.dom._
 import javax.xml.transform.sax._
 import javax.xml.transform.stream._
 
+import com.codahale.metrics.RatioGauge.Ratio
+import com.codahale.metrics.{JmxReporter, MetricRegistry, RatioGauge}
 import com.rackspace.com.papi.components.checker.handler.ResultHandler
 import com.rackspace.com.papi.components.checker.servlet._
 import com.rackspace.com.papi.components.checker.step.base.{Step, StepContext}
 import com.rackspace.com.papi.components.checker.step.results.Result
 import com.rackspace.com.papi.components.checker.util.IdentityTransformPool
 import com.rackspace.com.papi.components.checker.wadl.{StepBuilder, WADLDotBuilder}
-import com.yammer.metrics.scala.{Instrumented, Meter, Timer}
-import com.yammer.metrics.util.PercentGauge
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import nl.grons.metrics.scala.{InstrumentedBuilder, Meter, Timer}
 import org.apache.commons.codec.digest.DigestUtils.sha1Hex
 import org.w3c.dom.Document
 
+import scala.util.Try
+
 object Validator {
+  /** The application wide metrics registry. */
+  val metricRegistry = new com.codahale.metrics.MetricRegistry()
```

Comment:
That's a great question... for someone else. I suck at names.

Either of those sound fine.